core/CHANGELOG.md

```diff
@@ -9,10 +9,6 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 ### Bug Fixes
 
 * **angular:** do not create duplicate menuController instances ([#28343](https://github.com/ionic-team/ionic-framework/issues/28343)) ([fa78676](https://github.com/ionic-team/ionic-framework/commit/fa78676d57eb80655ee9447ffa07dcfdae0c6b2a)), closes [#28337](https://github.com/ionic-team/ionic-framework/issues/28337)
-
-
-### Features
```

Comment:
My commit for the large title should have been a "fix" not a "feature". Unfortunately, I only realized this after I merged the PR, so I needed to make the correction here.